packages/react-ai/src/ai-conversation-styles.css

```diff
@@ -114,15 +94,47 @@
 }
 
 .ai-actions-bar__button {
-  border-radius: 4px;
-  border: #aeb3b7 1px solid;
-  background: #fff;
   display: flex;
   width: 28px;
   height: 28px;
-  padding: 6px 12px;
   justify-content: center;
   align-items: center;
   gap: 4px;
   overflow: hidden;
 }
+
+.ai-field__container {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  gap: 8px;
+  height: 40px;
+}
+
+.ai-field__input {
+  align-self: stretch;
+  width: 800px;
+  overflow: hidden;
+  resize: none;
+}
+
+.ai-field__button {
```

Comment:
Doesn't need to be addressed in this PR but shouldn't these be prefixed with `ai-conversation`?